pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/xpath/internal/GetCommentOnFunction.java

```diff
@@ -57,6 +57,3 @@ public FunctionCall makeCallExpression() {
         };
     }
 }
-
-
```

Comment:
pmd does not check witespaces right? so this is an bug in check and and reason to use spot.